modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/indexer/ModelPreFilterContributorsHolder.java

```diff
@@ -25,7 +25,7 @@
 public interface ModelPreFilterContributorsHolder {
 
 	public Stream<ModelPreFilterContributor> stream(
-		String entryClassName, Collection<String> excludeIds,
```

Comment:
This was fully  intentional and i was aware the other order used there. My understanding is that alphabetical order should be used.

Reply:
ya, i guess i should have sorted all of the _other_ holders =)

Reply:
you indeed did it, thanks :) the longer story, the question behind my commit was:  is it consistency over correctness? is it cohesion over convention? Is it better to have all "wrong" over only something "right"?  Intuitively I would have chosen consistency like you did, but was anyways curious to see, how this works out.

Reply:
ya i actually talked to bchan a little bit and really it just depends on the situation. theres no blanket answer that applies to everything. in this case, the sorting/renaming i did was low effort and low risk, so i went for it

Reply:
if sorting/renaming the "wrong" old code would have been more effort/risk, then i think the proper approach would have been to make the new code "right", the way you originally had it, even though there was inconsistency with the other classes